.github/workflows/build.yml

```diff
@@ -0,0 +1,30 @@
+name: Gradle build
+
+on:
+  push:
+    branches: [ dev, master ]
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 40
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: DeLaGuardo/setup-graalvm@5.0
+        with:
+          graalvm: 22.0.0.2
+          java: java11
+          arch: amd64
+      - uses: actions/cache@v2
```

Comment:
Why do you insist on using manual caches?

Reply:
What is the alternative for them? https://github.com/burrunan/gradle-cache-action ?

Reply:
https://github.com/mipt-npm/visionforge/blob/c6258e72fb96e38c5428690a25a87b21bfaaa7a8/.github/workflows/build.yml#L20